core/src/main/java/org/apache/shiro/authz/permission/WildcardPermissionResolver.java

```diff
@@ -28,7 +28,41 @@
  * @since 0.9
  */
 public class WildcardPermissionResolver implements PermissionResolver {
+    boolean caseSensitive;
+    
+    /**
+     * Constructor to specify case sensitivity for the resolved premissions.
+     * @param caseSensitive true if permissions should be case sensitive.
+     */
+    public WildcardPermissionResolver(boolean caseSensitive) {
+        this.caseSensitive=caseSensitive;
+    }
+
+    /**
+     * Default constructor. 
+     * Equivalent to calling WildcardPermissionResolver(false)
+     * 
+     * @see WildcardPermissionResolver#WildcardPermissionResolver(boolean)
+     */
+    public WildcardPermissionResolver() {
+        this(false);
```

Comment:
We may want to consider returning `WildcardPermission.DEFAULT_CASE_SENSITIVE` in order to keep the defaults in one location

Reply:
@Claudenw see above comment.  It is a nit, but please comment one way or the other

Reply:
I can do it when merging, thanks for confirming though!